Shopping List/ShoppingListCollectionViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ShoppingListCollectionViewController.swift
+//  Shopping List
+//
+//  Created by Alex on 5/3/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
+
+class ShoppingListCollectionViewController: UICollectionViewController {
+    
+    let shoppingListController = ShoppingListController()
+     let reuseIdentifier = "ItemCell"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Register cell classes
+        self.collectionView!.register(UICollectionViewCell.self, forCellWithReuseIdentifier: reuseIdentifier)
```

Comment:
This line is what was causing your app to crash. When you use storyboards and add a reuse identifier in the attributes inspector, you are effectively registering the cell. Trying to register it here again causes the crash. I commented this line out in Xcode, and voila! your app started right up...